data/moves.ts

```diff
@@ -6511,7 +6511,7 @@ export const Moves: {[moveid: string]: MoveData} = {
 		self: {
 			onHit(source) {
 				for (const pokemon of source.foes()) {
-					if (!pokemon.volatiles['dynamax']) pokemon.addVolatile('torment');
+					if (!pokemon.volatiles['dynamax']) pokemon.addVolatile('torment', source, this.dex.getActiveMove('gmaxmeltdown'));
```

Comment:
Doesn't `addVolatile` already default to the current effect here? Even if it doesn't, it might be better to grab the move from the third argument to `onHit` instead of creating a new copy.

Reply:
I’m just doing it like how the other gmax moves do it, if it’s better to do it the other way then fixing the others should be a priority

Reply:
G-Max Finale does already do it the way I suggested but I'd say the other G-Max moves could be improved at some point. (I checked regular moves and the only other times we use `getActiveMove` are either from inside a condition or in certain specific cases where we need a fresh move.)

Reply:
Yeah, after this is handled I'll open another PR to fix them unless someone beats me to it